tyk-data-plane/values.yaml

```diff
@@ -0,0 +1,382 @@
+    #   Example using bitnami/redis
+    #   - tyk-redis-master.tyk.svc.cluster.local:6379
+    #   Example using bitnami/redis with sentinel
+    #   - tyk-redis.tyk.svc.cluster.local:26379
+    #   Example using bitnami/redis-cluster
+    #   - tyk-redis-redis-cluster.tyk.svc.cluster.local:6379
+
+    # Redis password
+    # If you're using Bitnami Redis chart (e.g. bitnami/redis) please input
+    # your password in the field below
+    pass: ""
+
+    # Enables SSL for Redis connection. Redis instance will have to support that.
+    # Default value: false
+    # useSSL: true
+
+    # If using "Redis Cluster" set enableCluster to true
+    # (e.g. if using bitnami/redis-cluster)
+    # enableCluster: true
+
+    # Enables sentinel connection mode for Redis. If enabled, provide both
+    # mandatory values for sentinelPass and masterName.
+    # enableSentinel: false
+
+    # Redis sentinel password, only required while enableSentinel is true.
+    # For bitnami/redis the same password as Redis above
+    # sentinelPass: ""
+
+    # Redis sentinel master name, only required while enableSentinel is true.
+    # For bitnami/redis typically redis-master
+    # masterName: ""
+
+    # By default the database index is 0. Setting the database index is not
+    # supported with redis cluster. As such, if you have enableCluster: true,
+    # then this value should be omitted or explicitly set to 0.
+    storage:
+      database: 0
+
+
+
+tyk-gateway:
+  ## Default values for tyk-gateway chart.
+  ## This is a YAML-formatted file.
+  ## Declare variables to be passed into your templates.
+  ## See Tyk Helm documentation for installation details:
+  ## https://tyk.io/docs/tyk-oss/ce-helm-chart/
+  ## Registry for all Tyk images - https://hub.docker.com/u/tykio
+
+  # nameOverride overrides the Chart name. It is truncated to 63 characters.
+  # Default value: tyk-gateway.name
+  nameOverride: ""
+
+  # fullnameOverride overrides App name. It is truncated to 63 characters.
+  # Default value: tyk-gateway.fullname
+  fullnameOverride: ""
+
+  gateway:
+    # The hostname to bind the Gateway to.
+    hostName: tyk-gw.local
+
+    tls:
+      # When true, sets the gateway protocol to HTTPS.
+      enabled: false
+      # When true, it will install the certificate present in the templates folder, set to false when using
+      # a custom TLS certificate to avoid overwriting yours
+      useDefaultTykCertificate: true
+      # The name of the secret which should contain the TLS certificate you want to use with the gateway deployment
+      secretName: tyk-default-tls-secret
+
+    # kind is type of k8s object to be created for gateway.
+    kind: Deployment
+
+    # replicaCount specifies number of replicas to be created if kind is Deployment.
+    replicaCount: 1
+
+    # containerPort for gateway
+    containerPort: 8080
+
+    image:
+      # image repository for Tyk Gateway
+      repository: docker.tyk.io/tyk-gateway/tyk-gateway
```

Comment:
i think we should use `tykio/tyk-gateway` in data-plane. @caroltyk wdyt?

Reply:
Modified.